extensions/amp-story-panning-media/amp-story-panning-media.md

```diff
@@ -0,0 +1,159 @@
+-->
+
+<figure class="centered-fig">
+  <amp-anim alt="amp-story-panning-media example" width="304" height="548" layout="fixed" src="https://raw.githubusercontent.com/processprocess/amphtml/panning-media-docs/extensions/amp-story-panning-media/img/amp-story-panning-media.gif">
+    <noscript>
+    <img alt="amp-story-panning-media example" src="https://raw.githubusercontent.com/processprocess/amphtml/panning-media-docs/extensions/amp-story-panning-media/img/amp-story-panning-media.gif" />
+  </noscript>
+  </amp-anim>
+</figure>
+
+The `amp-story-panning-media` component provides a way to pan and zoom an image between pages in [Web Stories](https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/?format=stories).
+
+## Usage
+
+Use the `amp-story-panning-media` to transition an image between pages. The component can animate between position and zoom.
+
+## Environment setup
+
+Add the following import to your AMP documents:
+
+```html
+<script async custom-element="amp-story-panning-media" src="https://cdn.ampproject.org/v0/amp-story-panning-media-0.1.js"><script>
+```
+
+### Configuration
+
+The `amp-story-panning-media` component accepts 1 descendant. This descendant must be an [amp-img](https://amp.dev/documentation/components/amp-img/?format=stories). Components associate with each other when they have a child with the same `src`.
+
+```html
+<amp-story-page>
+  <amp-story-grid-layer>
+    <amp-story-panning-media layout="fill" x="-10%" y="-20%" zoom=".8">
+      <amp-img layout="fill" src="assets/world-map.jpg" width="4000" height="3059" ></amp-img>
+    </amp-story-panning-media>
+  </amp-story-grid-layer>
+</amp-story-page>
+
+<amp-story-page>
+  <amp-story-grid-layer>
+    <amp-story-panning-media layout="fill" x="20%" y="30%" zoom="2">
+      <amp-img layout="fill" src="assets/world-map.jpg" width="4000" height="3059" ></amp-img>
+    </amp-story-panning-media>
+  </amp-story-grid-layer>
+</amp-story-page>
+```
+
+## Attributes
+
+### x (optional)
+
+Specifies the horizontal position in percentage.  
+Centered on 0 (default) with positive values moving the image to the right (50% centering left edge of image) and negative to the left (-50% centering right edge of image).
+
+### y (optional)
+
+Specifies the vertical position in percentage.
+Centered on 0 (default) with positive values moving the image down (50% centering top edge of image) and negative upward (-50% centering bottom edge of image).
+
+### zoom (optional)
+
+Specifies the level of zoom.
+The default is 1. This corresponds to the image fitting the viewport with [`layout=fill`](https://amp.dev/documentation/guides-and-tutorials/learn/amp-html-layout/?format=stories).
+A higher value scales the image up (zooms in). A lower value scales the image down (zooms out).
+
+### lock-bounds (optional)
+
+Prevents the image from panning beyond the viewport. When using `lock-bounds` `width` and `height` must be specified on the `amp-img` child.
+
+## Demos
```

Comment:
This section is a bit difficult. I would remove it and link out to the demos as described in the comments below.

Reply:
Sounds good. I moved the demos to a README.